pro/src/pages/Home/Offerers/Offerers.jsx

```diff
@@ -122,13 +127,42 @@ const Offerers = () => {
     )
   }
 
+  const removeSuccessParams = () => {
```

Comment:
(détail) Pas certain que success dans l'url soit si génant que ça ?

Reply:
je dois supprimer success car si on reload la page avec la params la pop in se reouvre